client/components/currency-information-for-methods/index.js

```diff
@@ -24,7 +24,7 @@ const ListToCommaSeparatedSentencePartConverter = ( items ) => {
 	return (
 		items.join( ', ' ) +
 		' ' +
```

Comment:
This line isn't needed, it adds an extra space before the `, and`.

Reply:
Removed it, thanks for the heads-up!